client/src/Client/pages/Passenger/OrderRide/OrderRide.tsx

```diff
@@ -406,7 +432,7 @@ const OrderRide = () => {
           {isOrderRideLoading ? 'טוען...' : 'הזמינו נסיעה'}
         </Button>
       </form>
-    </div>
+    </CustomFontSizeContainer>
   );
```

Comment:
I am not sure I understand how CusttomFontSizeContainer fits here.. But I belive the deleted div is the cause of the verify issue in the next file